developer/frameworks/articles/09-info-framework/01-the-info-framework-intro.markdown

```diff
@@ -0,0 +1,81 @@
+# The Info Framework
+
+The Info Framework provides enhances flexibility for retrieving, processing, 
+and displaying any information, regardless of where the information comes from. 
+The Info Framework can process data from the database, created 
+through process in memory, or retrieved from an external source.
+
+The Info Framework is a new feature @product-ver@ and currently has limited 
+functionality. The current version is the first set for obtaining and 
+displaying information from external systems or custom data models. The Info 
+framework is also very lightweight, and  does not require you to implement any  
+specific interface to use it. That means you can use it for an object even if 
+you don't have access to the source. 
+
+Rather than create a monolithic framework which requires all of the features to 
+be implemented for the sake of using any, it is a collection of loosely coupled 
+micro-frameworks. This lowers the learning curve and minimizes work for 
+developers.
+
+| Note: Liferay veterans may notice some similarities between the [Asset Framework](asset-framework-link) and Info 
+| Frameworks. The Info Framework doesn't yet provide the full array of features 
+| that Assets provide, but it has many similar characteristics and is simpler 
+| to use.
+
+## Using the Info Framework
+
+The Info Framework simplifies information handling across the board. You can 
+use it to customize your own applications, and it provides new ways to use 
+existing Liferay features. In previous versions of Liferay, features like 
+Display Pages and the Asset Publisher could only be used if you create an 
+`AssetEntry`. With the Info Framework you can define lists to display through 
+the Asset Publisher, and any entity that can be represented as a Java class can have a Display Page without creating an `AssetEntry` for that entity.
+
+To use the Info Framework, you create a `InfoItemRenderer` and an Information 
+List Provider by implementing `InfoListProvider`. You can use renderers and 
```

Comment:
Same as above. It's important to send the message that Info Framework provides several interfaces that developers can leverage for different purposes. All of them are optional, although there are also synergies among them.